docs/content/metric-stores/metrics.md

```diff
@@ -55,3 +55,60 @@ It exposes the following metric-specific tags:
 See
 [here](https://github.com/alibaba/feathub/blob/master/python/feathub/metric_stores/tests/test_prometheus_metric_store.py#L122)
 for example usages.
+
+## CountMap
+
+CountMap is a group of metrics that show the number of occurrence of each
+feature value. It has the following parameters:
+
+- window_size: Optional with 0 as the default value. The time range to compute
+  the metric. It should be zero or a positive time span. If it is zero, the
+  metric will be computed from all feature values that have been processed since
+  the Feathub job is created.
+
+It exposes the following metric-specific tags:
+
+- metric_type: "count_map"
+- window_size_sec: The value of the window_size parameter in seconds.
+
+See
+[here](https://github.com/alibaba/feathub/blob/master/python/feathub/metric_stores/tests/test_metric_store.py#L82)
+for example usages.
+
+## RatioMap
+
+RatioMap is a group of metrics that show the proportion of occurrence of each
+feature value. It has the following parameters:
+
+- window_size: Optional with 0 as the default value. The time range to compute
+  the metric. It should be zero or a positive time span. If it is zero, the
+  metric will be computed from all feature values that have been processed since
+  the Feathub job is created.
+
+It exposes the following metric-specific tags:
+
+- metric_type: "ratio_map"
+- window_size_sec: The value of the window_size parameter in seconds.
+
+See
+[here](https://github.com/alibaba/feathub/blob/master/python/feathub/metric_stores/tests/test_metric_store.py#L85)
+for example usages.
+
+## AverageSize
```

Comment:
Would it be more generic to add an `average` metric that can be applied to any numerical values, and let users derive AverageSize by using a combination of `size` and `average`?